weak_to_strong/datasets.py

```diff
@@ -124,7 +124,7 @@ def format_anthropic_hh(ex, rng):
 
 
 def format_cosmosqa(ex, rng):
-    true_answer = ex["answer" + str(ex["label"])]
+    true_answer = ex[f"answer{str(ex['label'])}"]
```

Comment:
In an f-string the explicit `str()` isn’t needed
```suggestion
    true_answer = ex[f"answer{ex['label']}"]
```